src/threads.cpp

```diff
@@ -178,14 +178,8 @@ void MapperClass::PathCollisionCheckTask() {
             current_set_point.z = 0.0;
         }
 
-        // Shift trajectory PCL so it passes along drone position
-        Eigen::Vector3d vec_traj_to_drone;
-        pcl::PointCloud<pcl::PointXYZ> point_cloud_traj_through_drone;
-        helper::SubtractRosPoints(robot_projected_on_traj, robot_position, &vec_traj_to_drone);
-        helper::ShiftPcl(point_cloud_traj, vec_traj_to_drone, &point_cloud_traj_through_drone);
-
         // Check if trajectory collides with points in the map
-        this->GetCollidingNodesPcl(point_cloud_traj_through_drone, &colliding_nodes);
```

Comment:
This is the effective change, instead of using the parallel line, just use the line that goes through the traversing waypoints